src/app/cube/home/components/splash/splash.component.html

```diff
@@ -1,12 +1,12 @@
 <div class="splash">
   <div class="splash__search">
-    <div class="search__text subtitle-text">
+    <div class="search__text subtitle-text" tabindex="0" id="pageContent">
       Effective cybersecurity curriculum at your fingertips.
     </div>
     <div class="search__input">
       <i class="far fa-search"></i>
       <input
-        aria-label="Search clark"
+        id="pageContent"
```

Comment:
What's happening here? Why does the input have the same id as the div above it? Additionally, why remove the aria-label?

Reply:
There is an aria-label attached to that input. I think it's on a different line and I don't know why that I did that.

Reply:
There is an aria-label there a little ways down the block.